web/src/HelpSearchBar.tsx

```diff
@@ -0,0 +1,114 @@
+import { InputAdornment } from "@material-ui/core"
+import { InputProps as StandardInputProps } from "@material-ui/core/Input/Input"
+import React, { ChangeEvent, KeyboardEvent, useState } from "react"
+import styled from "styled-components"
+import { ReactComponent as CloseSvg } from "./assets/svg/close.svg"
+import { ReactComponent as SearchSvg } from "./assets/svg/search.svg"
+import {
+  InstrumentedButton,
+  InstrumentedTextField,
+} from "./instrumentedComponents"
+import {
+  Color,
+  Font,
+  FontSize,
+  mixinResetButtonStyle,
+  SizeUnit,
+} from "./style-helpers"
+
+export function searchDocs(query: string) {
+  window.open(
+    `https://docs.tilt.dev/search?q=${encodeURI(query)}&utm_source=tiltui`
+  )
```

Comment:
let's replace the `encodeURI` call with the `URL` constructor, so the query string will be formatted correctly! this is my bad for making the wrong suggestion when we were pairing. 😅 

```suggestion
  const docsSearchUrl = new URL("https://docs.tilt.dev/search")
  docsSearchUrl.searchParams.set("q", query)
  docsSearchUrl.searchParams.set("utm_source", "tiltui")
  window.open(docsSearchUrl)
```